services/provider/server/consumer/api_updater.go

```diff
@@ -20,6 +20,8 @@ type APIUpdater interface {
 	ListStorageConsumers(namespace string) (*ocsv1alpha1.StorageConsumerList, error)
 	UpdateStorageConsumer(sc *ocsv1alpha1.StorageConsumer) error
 	GetSecret(name, namespace string) (*corev1.Secret, error)
+	GetConfigMap(name, namespace string) (*corev1.ConfigMap, error)
```

Comment:
We need to rebase on top of https://github.com/red-hat-storage/ocs-operator/pull/1430 now since the Interface accepts a context.

Reply:
I'll rebase in the end once #1430  is merged otherwise I'll have to rebase all the time that PR is modified.

Reply:
decided to rebase since lot was changing which impact my changes.